src/ansys/fluent/core/launcher/pim_launcher.py

```diff
@@ -245,6 +248,16 @@ def launch_remote_fluent(
 
     instance.wait_for_ready()
 
+    file_service = PimFileTransferService(pim_instance=instance)
+
+    local_sifile = f"sifile_{uuid.uuid4().hex}"
+
+    file_service.download_file("sifile.txt", os.path.join(os.getcwd(), local_sifile))
+
+    time.sleep(10)
```

Comment:
Can use a retry loop based on the exception beahviour instead of the hardcoded sleep. Something like below:
```
max_retries = ...
wait_time_between_retires = 1

for i in range(max_retries):
  try:
    <code>
    break
  except <observed exception(s)> as ex:
    if i == max_retries - 1:
      raise TimeoutError from ex
    sleep(wait_time_between_retires)
  
```

Reply:
Done. Thank you.